mover-syncthing/Dockerfile

```diff
@@ -0,0 +1,83 @@
+# Build the Syncthing image from Golang
+FROM registry.access.redhat.com/ubi8/go-toolset as builder
+USER root
+
+# source directory
+WORKDIR /workspace
+
+# latest release as of writing this
+ARG SYNCTHING_VERSION="v1.18.4"
+ARG SYNCTHING_GIT_HASH="951b0589522e13e14e8b40d33ea66de1e69c8b4e"
+
+# clone & cd
+RUN git clone \
+  --depth 1 \
+  --branch=${SYNCTHING_VERSION} \
+  https://github.com/syncthing/syncthing.git
+WORKDIR /workspace/syncthing
+
+# Make sure we have the correct Syncthing release
+RUN /bin/bash -c "[[ $(git rev-list -n 1 HEAD) == ${SYNCTHING_GIT_HASH} ]]"
+
+# Ensure that syncthing builds with boringcryptolib 
+ENV GOLANG_FIPS=1
```

Comment:
Is this needed?
My understand from reading https://developers.redhat.com/blog/2019/06/24/go-and-fips-140-2-on-red-hat-enterprise-linux#fips_140_2_validated_cryptography is that this would be used at runtime to force the use of fips libraries as opposed to autodetecting based on the system's FIPS mode.

Reply:
Nope, I just removed it.

Reply:
Nope, I just removed it